core/src/main/java/io/questdb/cutlass/http/processors/TextImportProcessor.java

```diff
@@ -169,6 +169,18 @@ public void onPartBegin(HttpRequestHeader partHeader) throws PeerDisconnectedExc
                 }
             }
 
+            DirectUtf8Sequence createTableChars = rh.getUrlParam(URL_PARAM_CREATE_TABLE);
+            if (createTableChars != null) {
+                try {
+                    boolean createTable = Booleans.parseBoolean(createTableChars);
```

Comment:
In other boolean query params we simply do something like `boolean createTable = !Utf8s.equalsNcAscii("false", rh.getUrlParam(URL_PARAM_CREATE_TABLE));`. We should do the same here without introducing `Booleans` and `BooleanException`.

Reply:
> ruthy i.e 'true' versus 'anything else in that field'

Yes, true is the default value for the force header query param, so this validation is sufficient.

For the create table param, it should be other way around since the default is false.